pyfstat/grid_based_searches.py

```diff
@@ -1132,17 +1145,15 @@ def __init__(
         self.output_file_header = self.get_output_file_header()
 
     def get_savetxt_fmt(self):
-        fmt = []
-        if "minStartTime" in self.keys:
-            fmt += ["%d"]
-        if "maxStartTime" in self.keys:
-            fmt += ["%d"]
-        fmt += helper_functions.get_doppler_params_output_format(self.keys)
-        fmt += ["%.16g", "%.16g", "%d"]  # for delta_F0, delta_F1, tglitch
-        fmt += ["%.9g"]  # for detection statistic
+        fmt = helper_functions.get_doppler_params_output_format(self.output_keys)
+        fmt["delta_F0"] = "%.16g"
+        fmt["delta_F1"] = "%.16g"
+        fmt["tglitch"] = "%d"
+        fmt[self.detstat] = "%.9g"
         return fmt
 
 
+# FIXME below here not fixed for [minStartTime,maxStartTime] removal yet
```

Comment:
TODO I'm not yet sure enough what those classes are supposed to be doing to fix them...